providers/base/units/audio/jobs.pxu

```diff
@@ -39,13 +39,19 @@ requires:
  device.category == 'AUDIO'
  package.name == 'alsa-base'
  package.name == 'gir1.2-gst-plugins-base-0.10' or package.name == 'gir1.2-gst-plugins-base-1.0'
- package.name == 'pulseaudio-utils'
+ package.name in ['pulseaudio-utils', 'pipewire']
 command:
-  audio_settings.py store --file="$PLAINBOX_SESSION_SHARE"/pulseaudio_settings
-  audio_settings.py set --device=pci --volume=50
-  gst_pipeline_test.py -t 2 'audiotestsrc wave=sine freq=512 ! audioconvert ! audioresample ! autoaudiosink'
-  EXIT_CODE=$?
-  audio_settings.py restore --file="$PLAINBOX_SESSION_SHARE"/pulseaudio_settings
+  if check_audio_deamon.sh ; then
+    show_default_audio_device.sh
```

Comment:
Some jobs that not use `pipewire_utils.py` will use this `show_default_audio_device.sh` such as `id: bluetooth/audio_record_playback`. However, it could be merge into `pipewire_utils.py`, if it is the policy that should be followed.

Reply:
What is the policy you refering to?

Reply:
If cert team would like to avoid using shell script in the future.

Reply:
If some jobs only use the functionality of `show_default_audio_device.sh`, I think we can still merge it into `pipewire_utils.py` and made it as one of the parameter, something like `pipewire_utils.py show`.

And Pierre and I have discussed about `pipewire_utils.py`, this case you mentioned just proved that you wrote a high-level utility script that can be used by other jobs. Excelent job! Please move it into `checkbox-support/checkbox_support/scripts`.

Reply:
I suggest to change shell script to python script. The reason is
1. It would be easier to know what this script is doing from the script name.
2. If there is the 3rd audio system, it would be easier to extend.

Reply:
I propose to **migrate** `show_default_audio_device.sh` to `show_default_audio_device.py`, not **merge** to `pipewire_utils.py`. And yes, they could move to `checkbox-support/checkbox_support/scripts`.

Reply:
I don't think show_default_audio_device needs to be in checkbox-support for 2 reasons:
1. wpctl is not a high-level utility script
2. it already been used in pipewire_utils & pipewire_test

Reply:
So put `pipewire_unitls.py` into `checkbox-support` and leave `show_default_audio_device.py` in `checkbox-provider-base` ?

Reply:
I'll suggest merge `show_default_audio_device` into `pipewire_unitls`, simply because both of them used `wpctl`, I see it as one of the utility of pipewire. But if you know there is other scenarios might be better to make it individual, that's also fine.

What is unitls stand for?

Reply:
If I move `pipewire_utils.py` to the checkbox_support, then I have to create other file just like `audio_settings.py` ? or do we have any better way to deal with it ?